vitest.config.mts

```diff
@@ -25,12 +25,45 @@
 /// <reference types="vitest" />
 
 import { defineConfig } from 'vitest/config'
+import path from 'path'
 
 export default defineConfig({
   test: {
-    include: ['**/__tests__/**/*.test.tsx'],
     globals: true,
-    environment: 'jsdom',
-    setupFiles: './vitest.setup.ts'
+    watchTriggerPatterns: [
+      {
+        // matches any file inside the __testfixtures__ directory
+        pattern: /^.*\/ui-codemods\/.*\/__testfixtures__\/.*$/,
+        // file is the full path. e.g.  /Users/MyUser/code/instructure-ui/packages/ui-codemods/lib/__node_tests__/__testfixtures__/colors.input.ts
+        testsToRun: (file, _match) => {
+          const dirName = path.basename(path.dirname(file))
+          // reruns all tests that match the directory name of the test fixture
+          return `packages/ui-codemods/lib/__node_tests__/${dirName}.test.ts`
+        }
+      }
+    ],
```

Comment:
This tricky code allows tests to rerun in watch mode when one makes changes to a fixture. For it to work the filename/directory conventions mentioned above must be kept.